### DISCUSSION TITLE
Need a feature to share the personal dailydev data

### DISCUSSION THREAD

**Discussion by dragon-whale** [dailydotdev/daily#660] [Feature Requests]
Say, I am working in a company and I am using the company mail to use daiylydev. Now, If I, one-day, resign from the company, my email will be taken away and all my stars and saved posts will be lost. So, we need a feature by which we can copy, share and paste the user data to another account. 

Please implement this feature as soon as you can.

**Comment by idoshamun** [comment]
Hi @abdullah-mohammad-khan,

We're working on delivering a new authentication system that will support adding multiple providers so you can easily achieve what you're looking for.
It will take a few weeks. I'll keep you posted

**Reply by dragon-whale** [reply]
@idoshamun Any update on this one ?

**Reply by idoshamun** [reply]
Yap. It's live now. You can go to your account profile and link more social providers
